docs/conf.py

```diff
@@ -329,4 +329,9 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'https://docs.python.org/': None}
+intersphinx_mapping = {
+    'flask': ('http://flask.pocoo.org/docs/latest/', None),
+    'invenio-access': ('https://pythonhosted.org/invenio-access', None),
+    'invenio-pidstore': ('https://pythonhosted.org/invenio-pidstore', None),
+    'python': ('https://docs.python.org/3.4', None),
```

Comment:
@hachreak normally we are still using 2.7 in other packages. Alternatively, you can use "latest" or 3.5.